### ISSUE TITLE
Pretrained model is here.

### ISSUE THREAD

**Issue by MaxMax2016** [PlayVoice/vits_chinese#3]
链接：https://pan.baidu.com/s/1CEgyC1R3FxXEI-5AL_Sj8g 
提取码：cym1

**Comment by OnceJune** [comment]
How many steps do you trained on Chinese dataset? Thanks in advance.

**Comment by dtx525942103** [comment]
3 GPU 16 batch_size about 200K step, you should focus loss:

baker_base	INFO	[200000, 0.00017334926433399393]
baker_base	INFO	loss_disc=2.410, loss_gen=2.418, loss_fm=6.987
baker_base	INFO	loss_mel=17.984, loss_dur=1.342, loss_kl=2.022
baker_base	INFO	Saving model and optimizer state at iteration 1143 to ./logs/baker_base/G_200000.pth
baker_base	INFO	Saving model and optimizer state at iteration 1143 to ./logs/baker_base/D_200000.pth

**Comment by josh-zhu** [comment]
谢谢@dtx525942103 提供的pretrained 模型，跑了下停顿感还是很强，是不是因为你提的 “音素后面强插边界了，VITS又强插边界  ” 的原因 ，要解决停顿感的问题是不是需要将add_blank 设置为 false 重新训练？期待回复，谢谢哈

**Comment by dtx525942103** [comment]
@josh-zhu 已经是False，你的音频和提供的样本差别大不大呢？

**Comment by josh-zhu** [comment]
好像没变化呢，时长都一致

**Comment by dtx525942103** [comment]
你把你的文字和音频放粘贴上来，我看看和我的状态一致不

**Comment by josh-zhu** [comment]
[vits_out.zip](https://github.com/dtx525942103/vits_chinese/files/8107000/vits_out.zip)
新进来的宝贝，赶紧去抢福利了哈

**Comment by josh-zhu** [comment]
1_baker_sample.wav 是和原有的vits_string.txt 保持一致的;感谢大佬及时回复哈

**Comment by dtx525942103** [comment]
你的ZIP是空的

**Comment by josh-zhu** [comment]
> 你的ZIP是空的

抱歉更新了哈

**Comment by dtx525942103** [comment]
这个效果很好了啊，正常效果啊

**Comment by josh-zhu** [comment]
oo,那就是没问题的;我和fastspeech2 的结果进行比较，感觉fastspeech2在语调上更自然;因为fastspeech2 + hifi 不是 e2e的，我还可以对每个phonme的mel 时长进行微调,整体更舒服一下;大佬方便留个线下联系的方式吗，做很多和语音相关的东西，想方便时请教交流下

**Comment by dtx525942103** [comment]
QQ525942103

**Comment by taotaoyuhust** [comment]
报错：
size mismatch for enc_p.emb.weight: copying a param with shape torch.Size([219, 192]) from checkpoint, the shape in current model is torch.Size([178, 192]).
请问是需要修改哪里啊
